nextflow.config

```diff
@@ -340,6 +345,7 @@ env {
     R_PROFILE_USER   = "/.Rprofile"
     R_ENVIRON_USER   = "/.Renviron"
     JULIA_DEPOT_PATH = "/usr/local/share/julia"
+    SYLPH_TAXONOMY_CONFIG = "./config.json"
```

Comment:
What is this and where does it come from? Does the user have to make this? It should be documented somehow if so

Reply:
This is solved in the modules level now, so I will remove it.